modules/ROOT/pages/dw-core-functions-filter.adoc

```diff
@@ -145,76 +145,6 @@ users filter ((item, index) -> item.error == null)
 ]
 ----
 
-=== Example
```

Comment:
Might need to add this example to the dw source. I assume this was added to the docs for a reason.